### ISSUE TITLE
Include motion switch for Protect Sensor

### ISSUE THREAD

**Issue by jaredfiorali** [hjdhjd/homebridge-unifi-protect#889]
**Is your feature request related to a problem? Please describe:**
I was hoping to be able to configure my Protect Sensor so the motion sensor will activate when my alarm is set to Away, and it will deactivate when my alarm is set to Home.

**Describe the solution you'd like:**
I believe the simplest solution is to Include a motion switch for Protect Sensors. This allows the sensor to be used in other cases, making this implementation a bit more flexible.

With a motion switch I can activate/deactivate it when the alarm state changes, solving the original issue.

**Describe alternatives you've considered:**
It might also be possible to include a feature option to enable/disable sensors based off of alarm status, however that makes this solution less flexible for other users.

**Additional context:**
I'm assuming inserting [this code](https://github.com/hjdhjd/homebridge-unifi-protect/blob/ea1c878b621c5e4bbb6cf1c0f32ce8cd70793b9e/src/protect-accessory.ts#L212-L222) (or something similar) in to the [protect-sensor.ts file](https://github.com/hjdhjd/homebridge-unifi-protect/blob/ea1c878b621c5e4bbb6cf1c0f32ce8cd70793b9e/src/protect-sensor.ts#L74) would work, but I'm not 100% certain.

**Comment by hjdhjd** [comment]
Thanks - I'll add this to the backlog for a future release! Great suggestion.

**Comment by jaredfiorali** [comment]
No problem at all @hjdhjd! I'm Happy to contribute a suggestion to a fantastic product 😄 

Just an FYI for anyone who might want to achieve the functionality I mentioned with the current software. I was able to integrate the protect sensor with the following steps:
- Disable the alarm feature for homebridge-unifi-protect
- Enable motion switches in homebridge-unifi-protect
- Install [homebridge-securitysystem](https://github.com/MiguelRipoll23/homebridge-securitysystem)
- Create scenes which enable/disable certain cameras depending on home presence (home, away, etc)
- Create a motion trigger for the protect sensor, only when the alarm system is set to away. I had to do this with Eve+ as it allows for conditional automations.

Obviously it would be better to have it configured natively in this homebridge-unifi-protect app, but this works for now 👍
